app/service/task_status_service.rb

```diff
@@ -85,9 +85,11 @@ def support_contact_details_status
 
   def make_live_status
     if @form.has_draft_version
-      return mandatory_tasks_completed? ? :not_started : :cannot_start
```

Comment:
Can we refactor this to:

```ruby
  def declaration_status
    return :completed if @form.declaration_section_completed
    return :in_progress if @form.declaration_text.present?

    :not_started
  end
```

https://refactoring.guru/replace-nested-conditional-with-guard-clauses

Reply:
I wasn't sure whether you meant this for the method I'd edited or the `declaration_status` method, so I just refactored all the methods

Reply:
🤩 thank you!